.editorconfig

```diff
@@ -1,136 +1,188 @@
-﻿###############################
```

Comment:
Did you copy this file from somewhere or did you make individual choices on all these?

Reply:
This is copied from azure-sdk-for-net:
https://github.com/Azure/azure-sdk-for-net/blob/main/.editorconfig

Reply:
With a change to `var` vs explicit type names.   azure-sdk-for-net set the preference to explicit types, but set the notification level to `none`.  I believe this would only affect code generated during refactoring, as the IDE wouldn't recommend changing existing code.

I changed this to prefer `var` with notification level `suggestion`.   Most of the time, the exact type is either unimportant or can be inferred from its assignment or usage.

Reply:
Lol in our next 1:1 I can give you some of the history on the var rule but in general I'm only in favor of var if the type name is easily discoverable by the one line of code.

Reply:
As for the rules set I was just curious about whether or not you stated with a wellknown setup which it sounds like you have so I'm good with that and we can always iterate on them as needed.